packages/frontend/src/help/logics/petri-net.mdx

```diff
@@ -0,0 +1,46 @@
+import { A } from "@solidjs/router";
+import { HelpAnalysisById } from "../logic_help_detail"
+
+## Description
+
+**Petri nets** are traditionally a way of expressing discrete-event dynamical systems with concurrency, or networks of resources and processes, describing how various states (called **places**) relate to one another in terms of **transitions**.
+Each transition has incoming and outgoing arrows connected to places, which describe the **input** and **output** places of the transition (respectively).
+
+One way of understanding Petri nets is through their **token semantics**.
+We place a number of "tokens" in each place and then check all the transitions to see if all their input places have a token: if so, then we can "fire" the transition, moving the tokens from the input places to the output places; if not, then nothing happens.
+In many classical references the token semantics are built directly in to the definition of Petri nets (often called a **marking** or **tokening**), but from a categorical point of view --- and, in particular, from the CatColab point of view --- the underlying shape of Petri nets is itself an expressive formal gadget.
```

Comment:
As a matter of style, I don't think we should call unnecessary attention to the "categorical point of view" (even if we like it). I'd just say something like "but an unmarked Petri net is already a useful formal gadget."